.github/build.go

```diff
@@ -118,6 +118,21 @@ func main() {
 		fmt.Println("-", m.String())
 	}
 
+	if os.Getenv("SKIP_PACKAGES") != "" {
+		for _,skip := range strings.Split(os.Getenv("SKIP_PACKAGES"), " ") {
+			for index, pkg := range missingPackages {
+				name := fmt.Sprintf("%s/%s", pkg.Category, pkg.Name)
+				if name == skip {
+					fmt.Println("- Skipping build of package due to SKIP_PACKAGES: ", pkg.String())
+					// how absurd is this just to pop one element from a slice ¬_¬
```

Comment:
Just a nit on the style, so feel free to ignore it. It is more common instead to recreate a new slice and omitting the ones that should be skipped while repopulating it. There is a bit of payoff on memory, but it is more readable

Reply:
Well, I only see a problem with that approach.

I dont know how to do it any other way :rofl:

Reply:
That's not really _so_ important, but here it is a small example: https://play.golang.org/p/2L2WPk9kR6E